pkg/controller/master/storageclass/storage_class_controller.go

```diff
@@ -131,9 +132,11 @@ func (h *storageClassHandler) syncStorageProfile(sc *storagev1.StorageClass) err
 	profileName := sc.Name
 	profile, err := h.storageProfileCache.Get(profileName)
 	if err != nil {
+		// StorageProfile doesn't exist yet. The CDI controller will automatically create
+		// one for each StorageClass, so we'll retry in a moment to let that happen.
```

Comment:
CDI create storage profile implicitly once a storage class is created
[1]: CDI watches storage class events https://github.com/kubevirt/containerized-data-importer/blob/3cfe2b73b232d32ba75ba06cf0f80080dd4663ef/pkg/controller/storageprofile-controller.go#L418
[2]: Create storage profile if not exists https://github.com/kubevirt/containerized-data-importer/blob/3cfe2b73b232d32ba75ba06cf0f80080dd4663ef/pkg/controller/storageprofile-controller.go#L148